pom.xml

```diff
@@ -478,6 +483,11 @@
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>com/redhat/red/build/finder/JSONUtils$*.class</exclude>
+          </excludes>
```

Comment:
Out of curiosity what is the exclude for?

Reply:
There are some anonymous inner classes in there that the test coverage is complaining are not tested. I really don't mind either way if this stays or goes. If it's actually a false positive, I think it would make sense to leave the change, and otherwise it should be removed.

Reply:
Ok cool :-) Understand now.